Windows 10: `imports-loader` not triggered for Globalize.js

**Bug**

When building applications on Windows 10, the `imports-loader` is not triggered for Globalize.js modules, resulting in `Can't resolve {module}` errors. For example,

```
ERROR in ./~/globalize/dist/globalize/date.js
Module not found: Error: Can't resolve 'cldr/supplemental' in 'XXX/node_modules/globalize/dist/globalize'
 @ ./~/globalize/dist/globalize/date.js 22:2-28:14
 @ ./~/globalize/dist/node-main.js
 @ ./src/main.js
```

--- a/src/webpack.config.ts
+++ b/src/webpack.config.ts
@@ -255,7 +255,7 @@ function webpackConfig(args: Partial<BuildArgs>) {
 					}
 				]},
 				{ test: /\.js?$/, loader: 'umd-compat-loader' },
-				{ test: /globalize(\/|$)/, loader: 'imports-loader?define=>false' },
+				{ test: new RegExp(`globalize(\\${path.sep}|$)`), loader: 'imports-loader?define=>false' },
 				...includeWhen(!args.element, () => {
 					return [
 						{ test: /\.html$/, loader: 'html-loader' }

